File: applicatie/src/main/java/com/vdzon/administratie/UpdateMongo.java
package com.vdzon.administratie;

import com.mongodb.Mongo;
import org.mongeez.Mongeez;
import org.springframework.core.io.ClassPathResource;

/**
 * Created by robbert on 13/05/2016.
 */
public class UpdateMongo {

    public void start(Mongo mongo, String dbName){
//        MongoClient mongo = MongoClientFactory.maakMongoClient(config, MongoClient.getDefaultCodecRegistry());
        Mongeez mongeez = new Mongeez();
        mongeez.setFile(new ClassPathResource("dbscripts/mongeez.xml"));
        mongeez.setMongo(mongo);
        mongeez.setDbName(dbName);
        mongeez.process();

    }
}

File: storage/impl/src/main/java/com/vdzon/administratie/database/UpdateMongo.kt
package com.vdzon.administratie.database

import com.github.mongobee.Mongobee
import com.github.mongobee.exception.MongobeeException
import com.mongodb.MongoClient

class UpdateMongo {

    @Throws(MongobeeException::class)
    fun start(mongoClient: MongoClient, dbName: String) {
        Mongobee(mongoClient)
                .setChangeLogsScanPackage("com.vdzon.administratie.mongo.changelogs")
                .setDbName(dbName)
                .execute()
    }
}
